src/app/sidebar/components/TopBar.tsx

```diff
@@ -14,18 +14,16 @@ export interface TopBarProps {
 
 export function TopBar({ selectedSubject, handleTopBarBackClick }: TopBarProps): JSX.Element {
   return (
-    <HStack bg="white" p="1em" top="0" m="0" boxShadow="md" zIndex={500}>
```

Comment:
We don't want the entire topbar clickable given that's here we're going to likely put features. Also if it is a button, we need to make it more "clickable" anyhow.